### PR TITLE
Clean up detect-changes.yaml

### PR DESCRIPTION
**Pull request by djorgensendk**

### CODE REVIEW

File: source/Logging/source/Logging.Tests/Logging.Tests.csproj [hunk 1]
```diff
@@ -32,7 +32,7 @@ limitations under the License.
         <PrivateAssets>all</PrivateAssets>
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       </PackageReference>
-      <PackageReference Include="coverlet.collector" Version="3.1.0">
+      <PackageReference Include="coverlet.collector" Version="3.1.1">
```
Comment (dstenroejl) [inline source/Logging/source/Logging.Tests/Logging.Tests.csproj:35]: No no no @djorgensendk

File: source/Messaging/source/Messaging.Tests/Messaging.Tests.csproj [hunk 2]
```diff
@@ -42,7 +42,7 @@ limitations under the License.
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="coverlet.collector" Version="3.1.0">
+    <PackageReference Include="coverlet.collector" Version="3.1.1">
```
Comment (dstenroejl) [inline source/Messaging/source/Messaging.Tests/Messaging.Tests.csproj:45]: Nope

**Review by dstenroejl — COMMENTED** [review]
Added two comments

File: source/Logging/source/Logging.Tests/Logging.Tests.csproj (hunk 1 above)
Comment (djorgensendk) [reply source/Logging/source/Logging.Tests/Logging.Tests.csproj]: hovsa. Undone

File: source/Messaging/source/Messaging.Tests/Messaging.Tests.csproj (hunk 2 above)
Comment (djorgensendk) [reply source/Messaging/source/Messaging.Tests/Messaging.Tests.csproj]: Undone

**Review by dstenroejl — APPROVED** [review]

**Pull request merged**